pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -43,6 +43,7 @@ import (
 	_ "k8s.io/client-go/kubernetes"
 )
 
+// ResourceResyncTime is interval of synchronizing resources
```

Comment:
ResourceResyncTime defines the resync period of the garbage collector's informers.